solutions/swb-reference/src/environment-files/bootstrap.sh

```diff
@@ -85,15 +66,41 @@ ln -s "${FILES_DIR}/bin/mount_s3.sh" "/usr/local/bin/mount_s3.sh"
 printf "%s" "$S3_MOUNTS" > "/usr/local/etc/s3-mounts.json"
 echo "Finish mounting S3"
 
+OS_VERSION=`cat /etc/os-release | grep VERSION= | sed 's/VERSION="//' | sed 's/"//'`
```

Comment:
If we decide that AL1 isn't required for swb v2, we can remove this line and the branch for AL1 below, right?

Reply:
I'll keep just to make sure we're dealing with AL2